plugin/evm/atomic/atomictest/shared_memories.go

```diff
@@ -0,0 +1,85 @@
+package atomictest
+
+import (
+	"testing"
+
+	"github.com/ava-labs/avalanchego/chains/atomic"
+	"github.com/ava-labs/avalanchego/database/memdb"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/snowtest"
+	"github.com/stretchr/testify/assert"
+)
+
+type SharedMemories struct {
+	ThisChain   atomic.SharedMemory
+	PeerChain   atomic.SharedMemory
+	thisChainID ids.ID
+	peerChainID ids.ID
+}
+
+func (s *SharedMemories) AddItemsToBeRemovedToPeerChain(ops map[ids.ID]*atomic.Requests) error {
+	for _, reqs := range ops {
+		puts := make(map[ids.ID]*atomic.Requests)
+		puts[s.thisChainID] = &atomic.Requests{}
+		for _, key := range reqs.RemoveRequests {
+			val := []byte{0x1}
+			puts[s.thisChainID].PutRequests = append(puts[s.thisChainID].PutRequests, &atomic.Element{Key: key, Value: val})
+		}
+		if err := s.PeerChain.Apply(puts); err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
This seems to be a bit unusual. We only use this once, and the values that are populated aren't actually valid.

Reply:
It seems we don't use any value for remove requests, so not sure what to put there. I tried to remove this function and move it where we call it, but it was kind-of violated the sharedMemories (by skipping the sharedMemories but still using the `ThisChainID` and `PeerChain` etc). I can try to apply any suggestion, but can't think a better way.